src/assets/styles.js

```diff
@@ -0,0 +1,28 @@
+import { StyleSheet } from 'react-native';
+import colors from './colors';
+import metrics from '../themes/metrics.js'
+
+export const standardMargin = 16;
+export const standardPadding = 16;
+export const standardFontSize = 14;
+
+export const standardStyle = StyleSheet.create({
+    standardStyle: {
```

Comment:
Change this to something more meaningful. It looks like it's only used in large text so call it largeText.